onnxruntime/core/providers/tensorrt/tensorrt_execution_provider.cc

```diff
@@ -1626,7 +1626,11 @@ common::Status TensorrtExecutionProvider::Compile(const std::vector<FusedNodeAnd
             for (int j = 0, end = nb_dims; j < end; ++j) {
               dimensions.d[j] = static_cast<int32_t>(tensor_shapes[j]);
             }
-            trt_context->setBindingDimensions(binding_index, dimensions);
+            const bool status = trt_context->setBindingDimensions(binding_index, dimensions);
+            if (not status) {
```

Comment:
can you make the suggested cpplint change?
use operator ! instead of not. thanks.

Reply:
@jywu-msft sure. fixed.